src/main/java/at/hannibal2/skyhanni/features/fishing/tracker/FishingProfitTracker.kt

```diff
@@ -244,7 +245,8 @@ object FishingProfitTracker {
     }
 
     private fun tryAddItem(internalName: NeuInternalName, amount: Int, command: Boolean) {
-        if (!FishingApi.isFishing(checkRodInHand = false)) return
+        val isFightingRareSeaCreature = SeaCreatureFeatures.lastRareSeaCreatureDespawn.passedSince() < 35.seconds
```

Comment:
there is no coalition between time last rare sea creature died and if the player was fighting it as well.